docs/04-operation-guides/operations/02-install-kyma.md

```diff
@@ -112,7 +112,7 @@ To deploy Kyma with only specific components, run:
   ```yaml
 prerequisites:
   - name: "cluster-essentials"
-  - name: "istio-configuration"
+  - name: "istio"
```

Comment:
we should also add `istio-resources` to this list. (it's a regular component though)

Reply:
The list is just an example showing how to install specific components, so I guess we do not need to add it here.